corehq/apps/translations/models.py

```diff
@@ -92,3 +100,21 @@ def get_translations(cls, lang, key=None, one=False):
                 return dict([(key, val[0]) for key, val in translations.items()])
             else:
                 return translations
+
+
+class TransifexBlacklist(models.Model):
+    domain = models.CharField(max_length=255)
+    app_id = models.CharField(max_length=32)
+    module_id = models.CharField(max_length=32)
+    field_type = models.CharField(
+        max_length=100,
+        choices=(
+            ('detail', 'Case Detail'),
+            ('list', 'Case List'),
+        )
+    )
+    field_name = models.CharField(max_length=255)
+    display_text = models.CharField(max_length=255)
```

Comment:
this may not be needed, but if it is, it should probably change to a TextField

Reply:
any reason for using `field_name` instead of `case_property`?
`display_text` is not neeeded

Reply:
display_text was needed because you can have multiple properties displayed in different ways. For example due list displays `parent/parent/dob` as date of birth and the age in years. Ideally we'd use a different property in the module to filter those out, but at this time display_text is all that's passed to this.